### PR TITLE
COMMERCE-12013 Support CPAttachmentFileEntries deletion through API

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/COMMERCE-12013

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -1973,6 +1973,79 @@ info:
...
+                  schema:
+                      description:
+                          The resource external reference code
+                      example: AB-34098-789-N
+                      minLength: 1
+                      type: string
+                  style: simple
+            responses:
+                204:
+                    description:
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/rest-openapi.yaml:1993]: @dsitu actually the pattern for the delete method is this https://github.com/liferay/liferay-portal/blob/6537b29aa21c90775cea4537338a34842971e14d/modules/apps/headless/headless-admin-content/headless-admin-content-impl/rest-openapi.yaml#L444C1-L449C40 . 
Also please move the yaml file changes to a separate commit and the commit message finishes with _(yaml file)_

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by dsitu** [comment]
ci:stop

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
Add automation and resent on https://github.com/liferay-commerce/liferay-portal/pull/4218

**Comment by jaydawu** [comment]
ci:close

**Pull request closed without merging**